src/context/auth/auth.context.helper.js

```diff
@@ -28,8 +68,39 @@ function checkIsTokenExsist() {
   return token ? true : false;
 }
 
-async function logout(){
+function getuserInfo() {
```

Comment:
Since you are defining this getUserInfo in multiple files, put it in utils folder and import from there.